src/plugins/discover/public/application/components/table/table.tsx

```diff
@@ -26,6 +28,7 @@ export function DocViewTable({
   const [fieldRowOpen, setFieldRowOpen] = useState({} as Record<string, boolean>);
   const [multiFields, setMultiFields] = useState({} as Record<string, string[]>);
   const [fieldsWithParents, setFieldsWithParents] = useState([] as string[]);
+  const showMultiFields = getServices().uiSettings.get(SHOW_MULTIFIELDS);
```

Comment:
quick first thought while reading the code, since the value of `SHOW_MULTIFIELDS` is read and used in `table.tsx`, I wonder if it is necessary to add this field to html templates and Angular code?